Gemfile

```diff
@@ -16,9 +15,7 @@ group :jekyll_plugins do
   gem 'jekyll_frontmatter_tests'
 end
 
-
-gem 'hash-joiner'
-gem 'safe_yaml'
-
 gem 'jemoji'
-gem 'pry'
+group :development do
+  gem 'pry'
+end
```

Comment:
Are you doing much debugging of the plugins? If not, I don't think you need to include pry at all.